src/bindings/rust/README.md

```diff
@@ -4,13 +4,28 @@
 - infos/provides =
 - infos/description =
 
-# Rust Bindings
+# Rust Bindings for Elektra
 
-Rust bindings for libelektra.
+<div align="center">
+  <!-- Crates version -->
+  <a href="https://crates.io/crates/elektra">
+    <img src="https://img.shields.io/crates/v/elektra.svg"
+    alt="Crates.io version" />
+  </a>
+  <!-- docs.rs docs -->
+  <a href="https://docs.rs/elektra">
+    <img src="https://img.shields.io/badge/docs-latest-blue.svg"
+      alt="docs.rs docs" />
+  </a>
+</div>
+
+_Elektra serves as a universal and secure framework to access configuration parameters in a global, hierarchical key database._
+
+For more information about elektra itself, visit the [website](https://libelektra.org).
```

Comment:
```suggestion
For more information about Elektra itself, visit the [website](https://libelektra.org).
```

We write it with a capital E, I'm not sure it's properly documented.

Reply:
It is an "Eigenname", so it is simply grammar that tells us to make a big "E".